src/components/quote/quote.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { TextStyles } from '../typography/typography-v2';
+import { theme } from '../layout/theme';
+
+interface QuoteProps {
+  children: ReactNode | ReactNode[];
+}
+
+const QuoteText = styled.p`
+  ${TextStyles.quote}
+  color: ${theme.palette.text.topline};
+  padding-left: 20px;
+  margin: 0;
+`;
+
+const Quote = ({ children }: QuoteProps) => {
+  return <QuoteText>{children}</QuoteText>;
+};
+
+export default Quote;
```

Comment:
Named exports are preferred over default exports. Some reasons why: https://stackoverflow.com/a/68665805/3141881

So please change it to `export const Quote`.

Reply:
I fully support this. You almost never need default exports. Sometimes I try it out to challenge my thoughts about this but it's much more powerful sticking to the named exports.